karaf/init/pom.xml

```diff
@@ -67,9 +53,7 @@
                         <Bundle-ClassPath>.,_brooklyn-tosca-karaf-patches-${project.version}.jar</Bundle-ClassPath>
             <Import-Package>org.apache.brooklyn.api.*</Import-Package>
             <DynamicImport-Package>*</DynamicImport-Package>
```

Comment:
I'm not keen on the Dynamic-Import myself, and unfortunately Karaf's not helpful at runtime about what exactly it is dynamically importing:
```
karaf@amp>imports -b 305
Package                                   | Version        | Optional | ID  | Bundle Name
-------------------------------------------------------------------------------------------------------
*                                         |                |          | 305 | brooklyn-tosca-karaf-init
org.apache.brooklyn.api.catalog           | [0.10.0,1.0.0) |          | 305 | brooklyn-tosca-karaf-init
org.apache.brooklyn.api.entity            | [0.10.0,1.0.0) |          | 305 | brooklyn-tosca-karaf-init
org.apache.brooklyn.api.internal          | [0.10.0,1.0.0) |          | 305 | brooklyn-tosca-karaf-init
org.apache.brooklyn.api.location          | [0.10.0,1.0.0) |          | 305 | brooklyn-tosca-karaf-init
org.apache.brooklyn.api.mgmt.classloading | [0.10.0,1.0.0) |          | 305 | brooklyn-tosca-karaf-init
org.apache.brooklyn.api.mgmt              | [0.10.0,1.0.0) |          | 305 | brooklyn-tosca-karaf-init
org.apache.brooklyn.api.policy            | [0.10.0,1.0.0) |          | 305 | brooklyn-tosca-karaf-init
org.apache.brooklyn.api.sensor            | [0.10.0,1.0.0) |          | 305 | brooklyn-tosca-karaf-init
org.apache.brooklyn.api.typereg           | [0.10.0,1.0.0) |          | 305 | brooklyn-tosca-karaf-init
```
I had hoped it would have listed the imports it had found necessary, but no such luck.

It might be worth some further experimentation to see if we can do without it but at least it seems to be working at present.